dao/src/test/java/org/thingsboard/server/dao/service/BaseTenantServiceTest.java

```diff
@@ -275,4 +323,374 @@ public void testSaveTenantWithIsolatedProfileInMonolithSetup() {
+        assertResourceIsDeleted(savedTenant, savedResource);
+        assertOtaPackageIsDeleted(savedTenant, savedOtaPackage);
+        Assert.assertNull(rpcService.findById(savedTenant.getId(), savedRpc.getId()));
+
+        tenantProfileService.deleteTenantProfile(TenantId.SYS_TENANT_ID, savedProfile.getId());
+    }
+
+    private void assertOtaPackageIsDeleted(Tenant savedTenant, OtaPackage savedOtaPackage) {
+        Assert.assertNull(
+                otaPackageService.findOtaPackageById(
+                        savedTenant.getId(), savedOtaPackage.getId()
+                )
+        );
+        PageLink pageLinkOta = new PageLink(1000);
+        PageData<OtaPackageInfo> pageDataOta = otaPackageService.findTenantOtaPackagesByTenantId(savedTenant.getId(), pageLinkOta);
+        Assert.assertFalse(pageDataOta.hasNext());
+        Assert.assertEquals(0, pageDataOta.getTotalElements());
+    }
+
+    private void assertResourceIsDeleted(Tenant savedTenant, TbResource savedResource) {
+        Assert.assertNull(resourceService.findResourceById(savedTenant.getId(), savedResource.getId()));
+        PageLink pageLinkResources = new PageLink(1000);
+        PageData<TbResourceInfo> tenantResources =
+                resourceService.findAllTenantResourcesByTenantId(savedTenant.getId(), pageLinkResources);
+        Assert.assertFalse(tenantResources.hasNext());
+        Assert.assertEquals(0, tenantResources.getTotalElements());
+    }
+
+    private void assertUserIsDeleted(Tenant savedTenant, User savedUser) {
+        Assert.assertNull(userService.findUserById(savedTenant.getId(), savedUser.getId()));
+        PageLink pageLinkUsers = new PageLink(1000);
+        PageData<User> users =
+                userService.findUsersByTenantId(savedTenant.getId(), pageLinkUsers);
+        Assert.assertFalse(users.hasNext());
+        Assert.assertEquals(0, users.getTotalElements());
+    }
+
+    private void assertTenantAdminIsDeleted(Tenant savedTenant) {
+        PageLink pageLinkTenantAdmins = new PageLink(1000);
+        PageData<User> tenantAdmins =
+                userService.findTenantAdmins(savedTenant.getId(), pageLinkTenantAdmins);
+        Assert.assertFalse(tenantAdmins.hasNext());
+        Assert.assertEquals(0, tenantAdmins.getTotalElements());
+    }
+
+    private void assertEdgeIsDeleted(Tenant savedTenant, Edge savedEdge) {
+        Assert.assertNull(edgeService.findEdgeById(savedTenant.getId(), savedEdge.getId()));
+        PageLink pageLinkEdges = new PageLink(1000);
+        PageData<Edge> edges = edgeService.findEdgesByTenantId(savedTenant.getId(), pageLinkEdges);
+        Assert.assertFalse(edges.hasNext());
+        Assert.assertEquals(0, edges.getTotalElements());
+    }
+
+    private void assertDashboardIsDeleted(Tenant savedTenant, Dashboard savedDashboard) {
+        Assert.assertNull(dashboardService.findDashboardById(
+                savedTenant.getId(), savedDashboard.getId()
+        ));
```

Comment:
try to use assertJ with an alias like 
`assertThat(dashboardService.findDashboardById(tenant.getId(), dashboard.getId())).as("dashboard").isNull();`